### PR TITLE
update text to applications page for consumers

### PR DESCRIPTION
**Pull request by j1joey**
ME-183726587

### CODE REVIEW

**Comment by markgoho** [comment]
@j1joey I think you also need the translation key/string added to the dc translation file

**Review by markgoho — DISMISSED** [review]

File: spec/domain/operations/individual/apply_aggregate_to_enrollment_spec.rb [hunk 1]
```diff
@@ -192,7 +192,7 @@
       input_params = {eligibility_determination: new_eligibility_determination}
       @result = subject.call(input_params)
       expect(@result.success).to eq "Aggregate amount applied on to enrollments"
-      expect(family.hbx_enrollments.to_a.first.applied_aptc_amount).not_to eq family.hbx_enrollments.last.applied_aptc_amount
+      expect(family.hbx_enrollments.to_a.first.applied_aptc_amount).to eq family.hbx_enrollments.last.applied_aptc_amount
```
[the remark this answers is not in the document]
Comment (j1joey) [reply spec/domain/operations/individual/apply_aggregate_to_enrollment_spec.rb:195]: This was actually a test that was causing trunk to fail, I thought it had to do with my ticket but I resolved it in another pr. Ill take it out of this one now that its not necessary and pull in the changes

File: components/financial_assistance/db/seedfiles/translations/en/ic/financial_assistance.rb [hunk 2]
```diff
@@ -162,6 +162,7 @@
   'en.faa.publish_error.second_error_message' => 'There is an error while submitting the application for assistance determination.',
   "en.faa.eligibility_go_to_my_account_message" => "<b>If you’re already enrolled in DC Health Link’s Individual & Family plan</b>, you’re finished! To see your plan information, select <b>GO TO MY ACCOUNT</b>.",
   "en.faa.application_for_coverage" => "Application for Coverage",
+  "en.faa.update_reminder" => "If you make changes to your %{year} coverage that should apply to your %{year2} coverage too, you will need to come back and update your %{year2} application. You can do this by copying your latest %{year} application and submitting it for %{year2}. "
```
Comment (jacobkagon) [inline components/financial_assistance/db/seedfiles/translations/en/ic/financial_assistance.rb:165]: Add a comma at the end of this line

**Review by jacobkagon — CHANGES_REQUESTED** [review]

**Review by jacobkagon — DISMISSED** [review]
Adds new paragraph on the application year page instructing applicants to update their 2023 applications if they made changes to their 2022 application. LGTM!

File: components/financial_assistance/app/views/financial_assistance/applications/index.html.erb [hunk 3]
```diff
@@ -7,6 +7,11 @@
       <div class="col-md-10">
         <h1 class="darkblue no-buffer"><%= l10n('faa.cost_savings_applications') %></h1>
         <h4><%= l10n('faa.cost_savings_applications_desc') %></h4>
+        <% if FinancialAssistanceRegistry.feature_enabled?(:iap_year_selection_form) %>
+          <% oe_year = HbxProfile.current_hbx.benefit_sponsorship.benefit_coverage_periods.last.open_enrollment_end_on.year %>
+          <% current_year = HbxProfile.current_hbx.benefit_sponsorship.benefit_coverage_periods.last.open_enrollment_start_on.year %>
+          <h4><b><%= l10n('faa.coverage_update_reminder', year: oe_year, year2: current_year) %></b></h4>
```
Comment (jacobkagon) [inline components/financial_assistance/app/views/financial_assistance/applications/index.html.erb:13]: @j1joey are the placement of the years correct? OE year will be the next year, while current year is this year.

Comment (jacobkagon) [reply components/financial_assistance/app/views/financial_assistance/applications/index.html.erb]: Updated the dates

**Review by jacobkagon — APPROVED** [review]
Updated the dates to reflect current year and following year accurately

**Pull request merged**